<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmEmpresas</class>
 <widget class="QDialog" name="FrmEmpresas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 75, 75, 148);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffff00;&quot;&gt;Gestión de Empresas.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtMensaje">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbladmin">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;ADMINISTRADOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(215, 215, 215);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>10</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Empresa</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEmpresa">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRutaBd">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>561</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>44</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ruta BD:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>8</y>
        <width>117</width>
        <height>406</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="botAnadir">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Añadir</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/add.png</normaloff>:/Icons/PNG/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="botSiguiente">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Siguiente</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/Next.png</normaloff>:/Icons/PNG/Next.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="botAnterior">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Anterior</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/Previous.png</normaloff>:/Icons/PNG/Previous.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="botBuscar">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/search.png</normaloff>:/Icons/PNG/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="botEditar">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Editar</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/edit.png</normaloff>:/Icons/PNG/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="botGuardar">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/Save.png</normaloff>:/Icons/PNG/Save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="botDeshacer">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Deshacer</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/undo.png</normaloff>:/Icons/PNG/undo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="botBorrar">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Borrar</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/delete.png</normaloff>:/Icons/PNG/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="botListados">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Listados</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/List.png</normaloff>:/Icons/PNG/List.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="botCerrar">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Cerrar</string>
         </property>
         <property name="icon">
          <iconset resource="terra.qrc">
           <normaloff>:/Icons/PNG/alert.png</normaloff>:/Icons/PNG/alert.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>89</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="terra.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>botCerrar</sender>
   <signal>clicked()</signal>
   <receiver>FrmEmpresas</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
